gandalf.js

```diff
@@ -1,27 +1,28 @@
 /**
```

Comment:
SCRIPT-The CSS rules here seem to be redundant. Can you please review and remove unnecessary ones?